### DISCUSSION TITLE
Remove things fron your home directory plz

### DISCUSSION THREAD

**Discussion by pablo-icm** [marbits-icm/marbits-public#11] [Announcements]
Hi everyone,

Some of your `/home` directories are at the maximum of your quotas. Could you please check them and see whether you can dele some of them?

Thanks!

**Comment by ramalok** [comment]
Also, maybe the home of some users that are not here anymore could be moved elsewhere or deleted?

**Reply by pablo-icm** [reply]
Sure they can. I'll move some
